repose-aggregator/tests/release-verification/src/vagrant/deb/Vagrantfile

```diff
@@ -0,0 +1,172 @@
+    version="=${1}"
+  fi
+fi
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Installing Repose ${name} package"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+if [ "X_${name}_X" = "X_local_X" ] ; then
+  dpkg -i /vagrant/repose-valve*_all.deb /vagrant/repose-filter-bundle*_all.deb /vagrant/repose-extensions-filter-bundle*_all.deb /vagrant/repose-experimental-filter-bundle*_all.deb
+  apt-get install -y -f
+else
+  apt-get install -y repose-valve${version} repose-filter-bundle${version} repose-extensions-filter-bundle${version} repose-experimental-filter-bundle${version}
+fi
+mkdir -p /etc/systemd/system/repose-valve.service.d && echo "[Service]\nEnvironment=\"JAVA_OPTS=-Xdebug -Xrunjdwp:transport=dt_socket,address=10037,server=y,suspend=n\"" > /etc/systemd/system/repose-valve.service.d/local.conf
+sed -i '/JAVA_OPTS/c\JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,address=10037,server=y,suspend=n"' /etc/sysconfig/repose
+SCRIPT
+
+$copy_configs = <<SCRIPT
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Copying configs"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+if [[ -d "/vagrant/etc_repose" ]] ; then
+    cp --force /vagrant/etc_repose/* /etc/repose/
+fi
+SCRIPT
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  #config.vm.box = "ubuntu/trusty64"
+  config.vm.box = "ubuntu/xenial64"
+
+  config.vm.provision "shell", inline: <<SCRIPT
+grep -q $(hostname) /etc/hosts
+if [ $? -ne 0 ]; then
+   echo '# Fixing a bug in some Xenial images where the local host name is not in the hosts file.' >> /etc/hosts
+   echo '# https://bugs.launchpad.net/ubuntu/+source/livecd-rootfs/+bug/1561250' >> /etc/hosts
+   echo "127.0.1.1 $(hostname)" >> /etc/hosts
+fi
+SCRIPT
+
+  # Expose the Repose Java Debug Wire Protocol (JDWP) port
+  config.vm.network :forwarded_port, guest: 10037, host: 18038
+  #config.vm.network "private_network", ip: "192.168.33.38"
+  # Expose the Repose port
+  config.vm.network :forwarded_port, guest: 8080, host: 18088
+
+  config.vm.provision "shell", inline: $update_yum
+  config.vm.provision "shell", inline: $install_repose_dependencies
+  #config.vm.provision "shell", inline: $other_tools
```

Comment:
This was a carry over from when this was a manual testing environment. Now that it is all automated and you shouldn't be editing the files within it, these were no longer needed.